deps/geth/config/keystore/.gitkeep

```diff
@@ -0,0 +1 @@
+These addresses are the same as the default accounts available in Orchestrate
```

Comment:
Is this some documentation about this directory?
Maybe we should rename as README.md so it can be more useful (and will still avoid git from removing the dir)

Reply:
renamed to readme